### PR TITLE
example: add hello-world ingress case

### PR DESCRIPTION
**Pull request by Xunzhuo**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Sub of #9

**Which issue(s) this PR fixes**:

- [ ] Update Quickstart Docs

Fixes

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by Xunzhuo** [comment]
xref: https://github.com/kubernetes-sigs/ingress2gateway/issues/21

File: examples/hello-world.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.k8s.io/v1
```
Comment (sunjayBhatia) [inline examples/hello-world.yaml:1]: should these examples also show the expected output of running the tool on the input Ingress?

Comment (Xunzhuo) [reply examples/hello-world.yaml:1]: Hey @sunjayBhatia. I think those are just examples we provided to have quickstart of i2gw.

Please see https://github.com/kubernetes-sigs/ingress2gateway/pull/18/files

**Comment by robscott** [comment]
@Xunzhuo let us know if you want to follow up on these, for now I'm going to close out in favor of more recent PRs.

**Pull request closed without merging**